language/tools/move-cli/src/experimental/cli.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{experimental, sandbox::utils::mode::Mode, Move};
+use anyhow::Result;
+use move_core_types::{
+    language_storage::TypeTag, parser, transaction_argument::TransactionArgument,
+};
+
+use structopt::{clap::arg_enum, StructOpt};
+
+#[derive(StructOpt)]
+pub enum ExperimentalCommand {
+    /// Perform a read/write set analysis and print the results for
+    /// `module_file`::`script_name`
```

Comment:
Extremely nit-picky, but just pointing out: some of the option docstrings in this pull request end with a period `.` and some don't. My preference would be to have all of them end with a period -- but please don't waste time on it if you don't mind either way :)

Reply:
Fixed! As long as we're consistent I'm happy :)